build-system/tasks/update-packages.js

```diff
@@ -95,6 +95,29 @@ function patchIntersectionObserver() {
   writeIfUpdated(patchedName, file);
 }
 
+/**
+ * TODO(samouri): remove this patch when a better fix is upstreamed (https://github.com/jakubroztocil/rrule/pull/410).
+ *
+ * Patches rrule to remove references to luxon. Even though rrule marks luxon as an optional dependency,
+ * it is used as if its a required one (static import). rrule relies on its consumers either
```

Comment:
super-nit: First `its` on this line should be `it's`.

Reply:
I appreciate the grammar fix :)